x/auth/keeper/params.go

```diff
@@ -21,10 +22,9 @@ func (ak AccountKeeper) SetParams(ctx sdk.Context, params types.Params) error {
 // GetParams gets the auth module's parameters.
 func (ak AccountKeeper) GetParams(ctx sdk.Context) (params types.Params) {
 	store := ctx.KVStore(ak.storeKey)
-	bz := store.Get(types.ParamsKey)
-	if bz == nil {
-		return params
+	params, err := store2.GetAndDecode(store, ak.decodeParams, types.ParamsKey)
```

Comment:
Now that we have store struct, cannot we better put it in the keeper struct?
Then we can just access it easily.

Reply:
I'm gonna do that, just trying to fix the tests first